### PR TITLE
Added serverless

### PR DESCRIPTION
**Pull request by shawju**
## what
* Added [serverless](https://github.com/serverless/serverless)

## why
* Need to use serverless to create Lambdas

### CODE REVIEW

**Review by bstascavage — DISMISSED** [review]

**Comment by bstascavage** [comment]
@shawju make sure you have your settings correct: https://github.com/saic-oss/anvil/pull/23#issuecomment-845970024

**Review by RothAndrew — CHANGES_REQUESTED** [review]
- Delete src/.DS_Store

Other than that it LGTM

**Comment by RothAndrew** [comment]
I'm reviewing from a compliance perspective only. Should get another approval from anvil team before merging

File: src/docker/Dockerfile
Comment (bstascavage) [inline src/docker/Dockerfile]: I don't think we want this ignore here.  We should update `wget` to use the `--progress` flag.

See [here](https://github-wiki-see.page/m/hadolint/hadolint/wiki/DL3047)

File: src/docker/Dockerfile
Comment (bstascavage) [inline src/docker/Dockerfile]: I don't think we want this ignore here.

See [here](https://github-wiki-see.page/m/hadolint/hadolint/wiki/DL3042)

File: src/docker/Dockerfile
Comment (bstascavage) [inline src/docker/Dockerfile:155]: What does this do?

**Review by bstascavage — CHANGES_REQUESTED** [review]
We shouldn't be adding `hadolint` ignores unless we explicitly need the acception.

**Review by bstascavage — APPROVED** [review]
LGTM

File: spec/skopeo_spec.sh
Comment (bstascavage) [inline spec/skopeo_spec.sh]: One last thing

This file needs to be indented like the other `shellspec` files

**Review by bstascavage — CHANGES_REQUESTED** [review]

**Review by RothAndrew — APPROVED** [review]
Approving from compliance perspective only. Get a 2nd from @saic-oss/anvil

**Review by bstascavage — APPROVED** [review]
LGTM

**Pull request merged**
